bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -738,6 +627,159 @@ public Image(Device device, ImageDataProvider imageDataProvider) {
 	init();
 }
 
+
+/**
+ * Upon zoom update, this method copies the image data and add disabled or grayed image depending on the flag
+ *
+ * @param device the device on which to create the image
```

Comment:
This documentation is incorrect, as the method does not create any image at all.

Reply:
I am kind of  confused here, shouldn't I just mention in docs what values should be passed to this method for correct usage?

Reply:
Yes, but the statement "the device on which to create the image" implies that the method creates an image, but it does nothing like that.